src/components/App/App.js

```diff
@@ -86,7 +86,7 @@ const App = ({ removeEventHandlers }) => {
 
         <Header />
         <ToolsHeader />
-        <div className="content">
+        <div className="content" onScroll={() => dispatch(actions.closeElement('annotationLineConnector'))}>
```

Comment:
Another place to put this line is

https://github.com/PDFTron/webviewer-ui/blob/2c7cfd4461285f9e1d7ef4931a72fd8c7294d62c/src/components/DocumentContainer/DocumentContainer.js#L181-L186

Might be better to put this line there for colocation.